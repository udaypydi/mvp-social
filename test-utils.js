// test-utils.js
import { render, queries } from '@testing-library/react'

const customRender = (ui, options) =>
  render(ui, { queries: { ...queries }, ...options })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }
