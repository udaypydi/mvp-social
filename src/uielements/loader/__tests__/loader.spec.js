import React from 'react';
import { render } from 'test-utils';
import  Loader from 'uielements/loader/loader.component.jsx';

it('should take a snapshot', () => {
    const { asFragment } = render(<Loader />)
    
    expect(asFragment(<Loader />)).toMatchSnapshot()
});
