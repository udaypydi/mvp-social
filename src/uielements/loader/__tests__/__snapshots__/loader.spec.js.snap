// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Loader--Container"
  >
    <div
      class="Loader--Element"
    />
  </div>
</DocumentFragment>
`;
